File: 1822. Sign of the Product of an Array/Solution.java
class Solution {
    public int arraySign(int[] nums) {
        int negs = 0;

        for(int n : nums){
            if(n == 0) return 0;
            if(n<0) negs++;
        }

        return negs % 2 == 0 ? 1 : -1;
    }
}

File: 1822. Sign of the Product of an Array/Solution.py
class Solution:
    def arraySign(self, nums: List[int]) -> int:
        negs = 0

        for n in nums:
            if n == 0 : return 0
            if n < 0 : negs += 1


        return 1 if negs % 2 == 0 else -1
